src/BinSkim.Rules/RuleIds.cs

```diff
@@ -48,8 +48,8 @@ internal static class RuleIds
 
         // Skipping some check namespace (BA3004-3009) for future checks
         public const string EnableReadOnlyRelocations = "BA3010";
+        public const string EnableBindNow = "BA3011";

```

Comment:
>EnableBindNow [](start = 28, length = 13)

Good names! Let's add test cases for this rule, @eddynaka.

Reply:
FYI there are some test cases for this already:

https://github.com/microsoft/binskim/blob/d9afb65c89a621411efded74c27999281d87867e/src/BuildSamples/ELF/build-test-binaries.sh#L59

https://github.com/microsoft/binskim/blob/d9afb65c89a621411efded74c27999281d87867e/src/BuildSamples/ELF/build-test-binaries.sh#L60

https://github.com/microsoft/binskim/blob/d9afb65c89a621411efded74c27999281d87867e/src/BuildSamples/ELF/build-test-binaries.sh#L73

Spoke offline with Eddy, and it may be helpful to update the no_immediate_binding test to explicitly use RTLD_LAZY binding: `-Wl,-z,lazy`

https://github.com/microsoft/binskim/blob/d9afb65c89a621411efded74c27999281d87867e/src/BuildSamples/ELF/build-test-binaries.sh#L74